chicagoland.pandemicresponsecommons.org/portal/gitops.json

```diff
@@ -1062,15 +1074,8 @@
   "useArboristUI": true,
   "showArboristAuthzOnProfile": true,
   "showFenceAuthzOnProfile": false,
-  "componentToResourceMapping": {
-    "Workspace": {
-      "resource": "/workspace",
-      "method": "access",
-      "service": "jupyterhub"
-    }
-  },
   "covid19DashboardConfig": {
-    "dataUrl": "https://opendata.datacommons.io/",
+    "dataUrl": "https://static.planx-pla.net/",
```

Comment:
put this back

Reply:
I need to tweak the automation config to skip that environment-specific setting.